examples/lines_bars_and_markers/fill_betweenx_demo.py

```diff
@@ -13,35 +14,35 @@
 x1 = np.sin(2 * np.pi * y)
 x2 = 1.2 * np.sin(4 * np.pi * y)
 
-fig, [ax1, ax2, ax3] = plt.subplots(3, 1, sharex=True)
+fig, [ax1, ax2, ax3] = plt.subplots(1, 3, sharey=True, figsize=(6, 6))
```

Comment:
Would this demo be helped by constrained_layout=True?

Reply:
Not in particular. The main issue is that the suplots were stacked vertically. This squeezes the y axis stongly so that it's hard to see what's going on. I just changed to stacking horizontally. This looks ok even without constrainedlayout. I've linked the result in the first post above.